pyanaconda/modules/network/installation.py

```diff
@@ -112,7 +114,8 @@ class NetworkInstallationTask(Task):
 """.strip()
 
     def __init__(self, sysroot, disable_ipv6, overwrite,
-                 network_ifaces, ifname_option_values):
+                 network_ifaces, ifname_option_values,
+                 configure_prefixdevname):
```

Comment:
I don't really agree with the variable name. I don't think we should use the same name in dracut which is inconsistent with the rest of the code here.

Reply:
Renamed